### DISCUSSION TITLE
Not able to find apps which use older SDK version

### DISCUSSION THREAD

**Discussion by bruno-espino** [aws/aws-sdk-java#2773] [Q&A] [answered]
Hello, AWS recommended via notification that we should update the AWS SDK for Java version but I'm not able to find which applications are affected via the AWS console, relevant guide: https://aws.amazon.com/blogs/developer/tls-1-3-incompatibility-with-aws-sdk-for-java-versions-1-9-5-to-1-10-31/

Maybe some of you faced this already?

**Comment by debora-ito** [comment, marked as answer]
Hi Yoruh, you won't find this information in AWS Console.

As the [blog post](https://aws.amazon.com/blogs/developer/tls-1-3-incompatibility-with-aws-sdk-for-java-versions-1-9-5-to-1-10-31/) describes, your application is impacted if you are using the AWS SDK for Java versions 1.9.5 to 1.10.31 to interact with AWS services. 

If you use `maven`, to identify the Java SDK version you're using you can run:
```
> mvn dependency:tree
```

**Comment by Ethanon** [comment]
Is there some good documentation on which versions of the SDK support which versions of TLS?
